cookbooks/tian_parameterization_kinematic_slab/coupled-two-phase-tian-parameterization-kinematic-slab.prm

```diff
@@ -0,0 +1,248 @@
+# This cookbook simulates a kinematically-driven 2D subduction system containing
+# distinct lithologies for the mantle wedge and subducting sediment,
+# mid-ocean ridge basalt (MORB), gabbro, peridotite. Lithologies within the slab
+# are prescribed an initial hydration state after a period of equilibration, and
+# subsequently dehydrate as they subduct. This cookbook utlizes the existing melt framework
+# within ASPECT which governs the mechanical coupling of a solid and fluid phase (see
+# https://doi.org/10.1093/gji/ggw329) but introduces solid-fluid reactions
+# (i.e partitioning of fluid into/out of the solid) that are governed by an approximation
+# published by Tian et al., 2019 (https://doi.org/10.1029/2019GC008488).
+# In detail, the model simulates the upper 20 km of a subducting slab with a dip
+# of 45 degrees and a convergence rate of 5 cm/yr. The Cartesian box is rotated
+# such that the x-axis aligns with the slab surface and the y-axis is orthogonal
+# to the slab surface, while gravity is imposed at an angle of 45 degrees. A surface pressure
+# of 1 GPa is imposed to account for the model simulating a portion of the slab
+# that is already subducted. For stability, the slab first subducts for 100 kyr before
+# the hydration is added, and the slab surface temperature diffuses to the mantle over
+# 10 km perpendicular to the slab surface. From the slab surface to the base of the model,
+# the composition is comprised of a 5km thick layer of sediment with 3 wt% water,
+# below is a 7km thick layer of MORB with 2 wt% water, and below that is an
+# 8 km thick layer of gabbro with 1 wt% water.
+
+set Adiabatic surface temperature              = 1600
+set Nonlinear solver scheme                    = iterated Advection and Stokes
+# We choose a low number of iterations and less restrictive nonlinear solver tolerance
+# for model efficiency. During the initial pulse of dehydration (for 3 time steps),
+# solvers do not quite converge to 1e-4, but achieve the convergence critera for the
+# rest of the model run time. For a production model, one would choose a tolerance
+# of 1e-5, or even stricter if possible.
+set Max nonlinear iterations                   = 25
+set Nonlinear solver tolerance                 = 1e-4
+set Dimension                                  = 2
+set End time                                   = 2e6
+set Surface pressure                           = 1e9
```

Comment:
It would be nice to add a comment here that you increase the surface pressure because the model starts deep in the mantle, not at the surface. Just because that is not a very common setup.